modules/version-update/pages/angular-js-UID-fork.adoc

```diff
@@ -0,0 +1,28 @@
+= Bonitasoft-managed AngularJs fork and integrated CVEs fixes
+:description: This page describes the AngularJs fork that is managed by Bonitasoft, as well as the cherry-picked CVE fixes from newer versions towards 1.3.x branch
+
+{description}
+
+[.card-section]
+== AngularJs fork
+As part of Bonitasoft's initiative to fix CVEs, an improvement was required in the UID Artifact builder.
+The current AngularJs version used by this part of UID is 1.3, and a migration to a newer version is impossible because of the breaking changes that exist in version 1.4. +
```

Comment:
```suggestion
The current AngularJS version used by this part of Bonita UI Designer is 1.3, and a migration to a newer version is impossible because of the breaking changes that exist in version 1.4. +
```